src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -1,3 +1,4 @@
+#include "n64dd.h"
```

Comment:
`#if PLATFORM_N64`?

Reply:
I kinda think we should drop this now, you haven't been following it yourself either (e.g. in the cic6105.c PR) which makes me think it's unnatural

Reply:
I haven't been following it because my workflow involves ignoring other versions while matching ntsc-1.2, I only add the #if s after the fact, and the rom matching for other versions doesn't catch these headers include

Reply:
Yeah, that's why I think we should include the headers anyway. There's less noise and I don't think the `#if` adds anything. But I don't think this PR is the place to change it, just bringing up my thoughts